### PR TITLE
Fix --affine CLI parameter

### PR DESCRIPTION
**Pull request by mstenta**
This proposes a fix for #243

Here is the contents of `00_stitcher.txt` with the following command after these changes:

Command: `sudo docker run --rm -v ./:/data openstitching/stitch --affine --verbose ./*.jpg`

Result `00_stitcher.txt`:

`AffineStitcher(**{'medium_megapix': 0.6, 'detector': 'orb', 'nfeatures': 500, 'matcher_type': 'affine', 'range_width': -1, 'try_use_gpu': False, 'match_conf': None, 'confidence_threshold': 1, 'matches_graph_dot_file': None, 'estimator': 'affine', 'adjuster': 'affine', 'refinement_mask': 'xxxxx', 'wave_correct_kind': 'no', 'warper_type': 'affine', 'low_megapix': 0.1, 'crop': True, 'compensator': 'no', 'nr_feeds': 1, 'block_size': 32, 'finder': 'dp_color', 'final_megapix': -1, 'blender_type': 'multiband', 'blend_strength': 5, 'timelapse': 'no'})`

### CODE REVIEW

File: stitching/cli/stitch.py [hunk 1]
```diff
@@ -315,6 +315,7 @@ def main():
     affine_mode = args_dict.pop("affine")
 
     if affine_mode:
+        args_dict.update(AffineStitcher.AFFINE_DEFAULTS)
         stitcher = AffineStitcher(**args_dict)
```
Comment (lukasalexanderweber) [inline stitching/cli/stitch.py:319]: please switch `AffineStitcher` to `Stitcher`, as the use of  `AffineStitcher` is not needed here so we can stay as generic as possible

**Review by lukasalexanderweber — COMMENTED** [review]
Could you plase also extend the CLI help text for the `--affine` parameter with what exactly is overwritten? So the users don't need to fallback to the tutorial:

`{'estimator': 'affine', 'wave_correct_kind': 'no', 'matcher_type': 'affine', 'adjuster': 'affine', 'warper_type': 'affine', 'compensator': 'no'}`

I also thought if it make sense to make a input validation for `AffineStitcher` that such a mistake can not happen again. E.g. that is validated that the kwargs don't diverge from the `AFFINE_DEFAULTS`. Such a validation would have made the CLI error immediately visible. 

What are your thoughts?

File: stitching/cli/stitch.py (hunk 1 above)
Comment (lukasalexanderweber) [reply stitching/cli/stitch.py:319]: ... but only if we DO NOT include a input validation for `AffineStitcher`

**Comment by mstenta** [comment]
This looks great @lukasalexanderweber! Thanks for picking it up! I didn't have time to dig into it last week. :+1:

**Pull request merged**